api/v2/openapi.yaml

```diff
@@ -53,6 +53,8 @@ paths:
           description: Get silences response
           schema:
             $ref: '#/definitions/gettableSilences'
+        '400':
+          $ref: '#/responses/BadRequest'
```

Comment:
this and the change in api/v2/api.go are the only changes as part of this PR - everything else is generated.